docs/l2-l3.rst

```diff
@@ -243,3 +243,45 @@ Delete Self IPs
 2. When prompted, click Confirm.
 
 The self IPs are deleted from the BIG-IP device, as well as from the application database.
+
+-------
+
+FAQ
+===
+
+**Q. Why do I get an error for VLAN/self IP delete operation from the App?**
+
+This is a known issue for BIG-IP v 12.x. If a pool with nodes is associated with a self IP of the same subnet, BIG-IP doesn’t allow user to delete that self IP. As a result, the VLAN delete operation also fails with the error.
+
+**Workaround:**
+
+- Delete the corresponding pool member from BIG-IP.
+
+- Perform the VLAN/self IP delete from App.
+
+- Recreate the pool member on BIG-IP.
+
+------
+
+**Q. When I try to stitch a VLAN tag, why do I see “VLAN not available for stitching”? Not able to configure the VLAN.**
+
+For a single BIG-IP device, after a VLAN tag is stitched for a particular logical device (say LDEV1), the same VLAN tag is not available for stitching again for a different Logical device (say LDEV2). This is because the VLAN tag is already present on the BIG-IP device and re-creating it for a different logical device is not allowed. In order to proceed with stitching, delete the original VLAN from the stitched LDEV, which is mentioned in this VLAN card’s info message. 
+
+.. note::
+   For a different BIG-IP login, this stitched VLAN tag will still be available for configuration. 
+
+------
+
+**Q. Why don’t I see the pre-existing BIG-IP VLANs and self IPs that have a different naming convention than the application?**
+
+The application does not support pre-existing VLANs that have a different naming convention than the app. It is able to detect VLANs that have been created and managed from the application only. 
+
+Although, after uninstalling and reinstalling the application, if the app database is lost, the application will be able to detect the previously created application VLANs by reading BIG-IP information and show them as Out-of-sync VLANs. The users will also be able to sync them to the application to rebuild App Database.
+
+The application displays APIC VLAN tags for a particular Logical Device Cluster on the L2-L3 stitching page. If there is an out-of-band VLAN with different naming convention but same VLAN tag on the BIG-IP device, the application detects it and shows it in the Out-of-sync information too. But the only action available for such a VLAN or self IP will be deletion of that object from BIG-IP. It cannot sync to application, since it has a different naming convention. The application also does not detect out-of-band information for any of the other VLAN tags that are not a part of APIC VLAN list. 
```

Comment:
...with a different naming convention but the same VLAN tag...
..from the BIG-IP.
It cannot sync to the application...